Packs/CommonScripts/Scripts/IsInternalDomainName/IsInternalDomainName.yml

```diff
@@ -0,0 +1,36 @@
+args:
+- default: false
+  description: The domain name that the sub domain should be in
+  isArray: false
+  name: domainName
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of domains to check.
+  isArray: true
+  name: domainToCheck
+  required: true
+  secret: false
+comment: Check if a Domain or sub domain is in a domain
+commonfields:
+  id: IsInternalDomainName
+  version: -1
+enabled: false
+name: IsInternalDomainName
+outputs:
+- contextPath: Domain.Name
+  description: The domain that was checked
+  type: String
+- contextPath: Domain.IsInternal
+  description: "True if sub domain is part of a domain. \nFalse otherwise"
```

Comment:
Shouldn't this also include the domain it was checked against?
@David-BMS WDYT?

Reply:
Done as David wrote in the issue